### ISSUE TITLE
Issue in the documentation

### ISSUE THREAD

**Issue by orboan** [jupyterhub/jupyterhub#3875]
Hello,

this is an issue, to my understanding, in the documentation, in particular in the README file in:

[service-announcement](https://github.com/jupyterhub/jupyterhub/tree/main/examples/service-announcement)

I already asked a fix for this README in discourse forum, but as I am getting no responses I decided to publish it here. My apologies if this is not the right place to submit issues in the documentation.

The problem is that this service is not working after I followed instructions in the README file.

I am getting:

```
[I 2022-04-25 12:41:07.987 JupyterHub service:132] Spawning /usr/bin/python3 -m announcement --log-level 10
/usr/bin/python3: No module named announcement
```

Maybe this is related with the fact that I should copy the `announcements.py` file to any path location, but this is not explained in the README file, so I do not know what to do with this file.

Maybe it is implicitly assumed that we should already know where to copy this file or how to properly deploy this service, but most of the times this is not the case. 

I understand documentation is something developers do not like a lot, but is one of the key factors that differentiate good software from regular software. And in general I found the documentation one of the less developed aspects, with a lot of room for improvement, in all jupyter-related projects.

Thanks

**Comment by rcthomas** [comment]
Maybe this example should be replaced with a link to the [more fully developed service](https://github.com/rcthomas/jupyterhub-announcement), similar to how cull-idle-servers was moved out?

**Comment by Temidayo32** [comment]
@orboan are you working on the issue? Or is there anyone working on it?

**Comment by akashthedeveloper** [comment]
Hey, @GeorgianaElena I would like to work on this issue can you please assign it to me?

**Comment by MrOrero** [comment]
Hi, @GeorgianaElena I am an Outreach applicant, I would like to work on this issue, I have already started working on it, can it be assigned to me?

**Comment by minrk** [comment]
@Temidayo32 are you still working on this? I can assign it to you, or let someone else take it if you are doing something else.

**Comment by Temidayo32** [comment]
@minrk Yeah, assign it to me.

**Comment by Achele** [comment]
Hi, @GeorgianaElena I am an Outreachy applicant, I would like to work on this issue.

**Comment by Temidayo32** [comment]
@minrk  Fixed this issue. Check out the commit https://github.com/jupyterhub/jupyterhub/commit/482c0548393ec7b6cb6dd21e8560ba89953c7ef7

**Comment by devanshi-jain** [comment]
Hey if this issue is done, could you close it pleasE?

**Comment by minrk** [comment]
@devanshi-jain thanks for noticing! Fixed by #4092
